dr/base/init.sls

```diff
@@ -84,14 +79,13 @@ install-{{pippkg}}:
     - sources:
       - salt://dr/home/bashrc
 
-/home/{{pillar.__stackdio__.username}}/.ssh/authorized_keys:
-  file:
-    - append
-    - template: jinja
-    - mode: 755
-    - makedirs: true
-    - sources:
-      - salt://dr/home/authorized_keys
+{% for sshkey in pillar.dr.authorized_keys %}
+users_authorized_keys.{{ loop.index }}:
+  ssh_auth:
+    - present
+    - user: {{ pillar.__stackdio__.username }}
+    - name: {{ sshkey }}
```

Comment:
Any benefit to using `names` here and just giving it the `pillar.dr.authorized_keys`?